### DISCUSSION TITLE
Does the VpnHood client support connection to an OpenVPN server and other ppt2p servers?

### DISCUSSION THREAD

**Discussion by Yarogleck** [vpnhood/VpnHood#517] [Q&A]
I apologize in advance for the superficial study of the project. I am in a hurry looking for a project that fully supports Managed client connection to any commercial VPN server, not through these TUN/TAP connections for the entire system at once and all that magic with writing packets to an empty network connection file.

**Comment by trudyhood** [comment]
Sorry, we don't support it.
